libqtile/backend/x11/window.py

```diff
@@ -997,7 +1000,6 @@ def focus(self, warp: bool) -> None:
                 self.window.set_property("_NET_WM_STATE", state)
 
         # re-grab button events on the previously focussed window
-        old = self.qtile.core._root.get_property("_NET_ACTIVE_WINDOW", "WINDOW", unpack=int)
```

Comment:
Why did this need to be moved?

Reply:
Ha. It doesn't any more. I moved it because I was using `old` in a test to limit warping but removed that test. I'll put it back.